aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -560,3 +560,4 @@ Dependency injection is an *alternative* to static/global object access patterns
 * [Explicit Dependencies Principle](https://deviq.com/explicit-dependencies-principle/)
 * [Inversion of Control Containers and the Dependency Injection Pattern (Martin Fowler)](https://www.martinfowler.com/articles/injection.html)
 * [New is Glue ("gluing" code to a particular implementation)](https://ardalis.com/new-is-glue)
+* [How to register a service with multiple interfaces in ASP.NET Core DI](https://andrewlock.net/how-to-register-a-service-with-multiple-interfaces-for-in-asp-net-core-di/)
```

Comment:
How to register --> Register

Reply:
Same thing ... the title is ...

> How to register a service with multiple interfaces in ASP.NET Core DI

Reply:
I need guidance (cc: @Rick-Anderson). I've been titling 3rd party *Additional resources* links by what the author of the content used as the title. Should I start changing the titles displayed in the docs in this section to conform to our titling rules? I don't recommend it, but I certainly will follow whatever rule is laid down.

Reply:
What's wrong with `## Additional resources` ?

Reply:
The question is should we use the titles exactly from pages that we link to as our link text or should we make their titles conform to our title guidelines in our link text?

Reply:
I wouldn't force them to conform to our style, but I sometimes change them up if they read better. It's rare we link to 3rd parties.

Reply:
Yeah ... I've been leaving them as the author has them. I would change a title in our link text if there was a spelling error or something along those lines. Thus far, 3rd party titles have been good.